ArduCopter/Copter.h

```diff
@@ -368,6 +368,7 @@ class Copter : public AP_Vehicle {
             uint8_t unused3                 : 1; // 25      // was compass_init_location; true when the compass's initial location has been set
             uint8_t unused2                 : 1; // 26      // aux switch rc_override is allowed
             uint8_t armed_with_switch       : 1; // 27      // we armed using a arming switch
+            uint8_t disarm_switch           : 1; // 28      // true if disarm aux switch is enable
```

Comment:
Do we need this state?

Can arming just check the channel `aux_switch_pos` instead?

Reply:
I've fixed to use aux_switch_pos_t.